deployment_config/prod_vars.yml

```diff
@@ -1 +1,8 @@
+AUTH_BASE: https://uat.hsesinfo.org
+AUTH_CLIENT_ID: ((AUTH_CLIENT_ID))
+AUTH_CLIENT_SECRET: ((AUTH_CLIENT_SECRET))
+env: prod
 instances: 2
+NODE_ENV: production
+REDIRECT_URI_HOST: https://tta-smarthub-prod.app.cloud.gov
```

Comment:
Probably not our eventual state, but give the current deployment config this would be the correct redirect uri.